README.md

```diff
@@ -32,6 +32,23 @@ In addition to the `admission` scenario, Gatekeeper's audit functionality allows
 
 Finally, Gatekeeper's engine is designed to be portable, allowing administrators to detect and reject non-compliant commits to an infrastructure-as-code system's source-of-truth, further strengthening compliance efforts and preventing bad state from slowing down the organization.
 
+## Admission Webhook Fail-Open Status
+
+Currently Gatekeeper is defaulting to using `Ignore` for admission request webhook errors. The impact of
+this is that when the webhook is down, or otherwise unreachable, constraints will not be
+enforced. Audit is expected to pick up any slack in enforcement by highlighting invalid
+resources that made it into the cluster.
+
+The reason for fail-open is because the webhook server currently only has one instance, which risks downtime
+during actions like upgrades. If we were to fail closed, this downtime would lead to
+downtime in the cluster's control plane. As the theoretical availability improves we
+will likely change the default to fail closed.
+
+If desired, the webhook can be set to fail closed by modifying the ValidatingWebhookConfiguration,
```

Comment:
nit: set to `failurePolicy: Fail`

Reply:
I don't want to presume that the API will be stable here. IMO this should be a high-level overview, not technical instructions.